WEB/Src/PerformanceCollector/PerformanceCollector/Perf.csproj

```diff
@@ -59,7 +59,7 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="System.Diagnostics.PerformanceCounter" Version="4.5.0" />
+    <PackageReference Include="System.Diagnostics.PerformanceCounter" Version="4.7.0" />
```

Comment:
is it possible to use >= so that we are resilient from future version bumps?

Reply:
I think we can specify a `4.*`
https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#references-in-project-files-packagereference

Reply:
Looks like this is just a compile-time rule. 4.* put 4.7.0 in the nuspec. 
`<dependency id="System.Diagnostics.PerformanceCounter" version="4.7.0" exclude="Build,Analyzers" />`